packages/kbn-ftr-common-functional-ui-services/services/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -427,16 +427,16 @@ export class WebElementWrapper {
   /**
    * Moves the remote environment’s mouse cursor to the current element with optional offset
    * https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/input_exports_Actions.html#move
-   * @param { xOffset: 0, yOffset: 0 } options
+   * @param { xOffset: 0, yOffset: 0, topOffset: number } options
    * @return {Promise<void>}
    */
-  public async moveMouseTo(options = { xOffset: 0, yOffset: 0, topOffsetOrOptions: 0 }) {
+  public async moveMouseTo({ xOffset = 0, yOffset = 0, topOffset = 0 }) {
```

Comment:
This change appeared to break a lot of tests. I would suggest reverting to the previous commit or initializing the arguments with something like `{ xOffset = 0, yOffset = 0, topOffset = 0 } = {}` (<-- I did not confirm this, so please double-check).

Reply:
Yeah I just caught it. I accidentally made it a required object parameter, but I just pushed a fix that makes it optional again